packages/whispers/PKGBUILD

```diff
@@ -2,17 +2,17 @@
 # See COPYING for license details.
 
 pkgname=whispers
-pkgver=1.5.3.r13.g5832bf7
+pkgver=2.1.0.r01
 pkgrel=2
-pkgdesc='Identify hardcoded secrets and dangerous behaviours.'
+pkgdesc='Identify hardcoded secrets in static structured text.'
 arch=('any')
 groups=('blackarch' 'blackarch-code-audit')
-url='https://github.com/Skyscanner/whispers/'
-license=('Apache')
+url='https://github.com/adeptex/whispers/'
+license=('GPL-3.0')
```

Comment:
change to `GPL3

its has to respect `/usr/share/licenses/common/` names as mentioned here https://wiki.archlinux.org/title/PKGBUILD#license